modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/test/java/com/liferay/portal/search/tuning/rankings/web/internal/index/name/RankingIndexNameBuilderImplTest.java

```diff
@@ -35,24 +35,25 @@ public class RankingIndexNameBuilderImplTest {
 
 	@Test
 	public void testMultiTenancy() {
-		_assertIndexName(
-			2021, companyId -> "liferay-" + companyId,
-			"liferay-2021-search-tuning-rankings");
-	}
-
-	private void _assertIndexName(
-		long companyId, IndexNameBuilder indexNameBuilder, String expected) {
-
 		RankingIndexNameBuilderImpl rankingIndexNameBuilderImpl =
 			new RankingIndexNameBuilderImpl();
 
 		ReflectionTestUtil.setFieldValue(
-			rankingIndexNameBuilderImpl, "_indexNameBuilder", indexNameBuilder);
+			rankingIndexNameBuilderImpl, "_indexNameBuilder",
+			new IndexNameBuilder() {
+
+				@Override
+				public String getIndexName(long companyId) {
+					return "liferay-" + companyId;
+				}
 
-		RankingIndexName rankingIndexName =
-			rankingIndexNameBuilderImpl.getRankingIndexName(companyId);
+			});
 
-		Assert.assertEquals(expected, rankingIndexName.getIndexName());
+		Assert.assertEquals(
+			"liferay-2021-search-tuning-rankings",
+			rankingIndexNameBuilderImpl.getRankingIndexName(
```

Comment:
@W2jx Please don't inline this, basically we don't allow chaining methods like `A.doOneThing().doAnotherThing()`.

Reply:
Ok, I will fix it.